src/models.yaml

```
@@ -0,0 +1,9 @@
+- UNet2DNucleiBroad:
+  config_url: https://raw.githubusercontent.com/bioimage-io/pytorch-bioimage-io/master/specs/models/unet2d/nuclei_broad/UNet2DNucleiBroad.model.yaml
```

These items should be indented another level, to go under `UNet2DNucleiBroad`, right?
```suggestion
    config_url: https://raw.githubusercontent.com/bioimage-io/pytorch-bioimage-io/master/specs/models/unet2d/nuclei_broad/UNet2DNucleiBroad.model.yaml
```

---

Just tried, the current syntax is correct, and if I try to add two more spaces, the code breaks.

---

What structure are you expecting? It's weird that we want a `None` value for the first key.